### PR TITLE
Feature/ch119096/subscription in clouds to process map views

### PR DESCRIPTION
**Pull request by juanrmn**
https://app.clubhouse.io/cartoteam/story/119096/subscription-in-clouds-to-process-map-views-stats-messages

### CODE REVIEW

File: app/models/concerns/cartodb_central_synchronizable.rb [hunk 1]
```diff
@@ -80,7 +80,7 @@ def allowed_attributes_from_central(action)
       case action
       when :create
         %i(name seats viewer_seats quota_in_bytes display_name description website
-           discus_shortname twitter_username geocoding_quota map_view_quota
+           discus_shortname twitter_username geocoding_quota map_views_quota
```
Comment (thedae) [inline app/models/concerns/cartodb_central_synchronizable.rb:83]: Just one question: if this field has been updated for the central sync process, shouldn't this modification also be applied in Central code?

File: lib/carto/subscribers/map_views_update.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+require './lib/carto/user_creator'
```
Comment (thedae) [inline lib/carto/subscribers/map_views_update.rb:1]: Do we need these dependencies? Looks like a copy&paste

Comment (thedae) [inline lib/carto/subscribers/map_views_update.rb:1]: Do we need these dependencies? Looks like a copy&paste

**Review by thedae — CHANGES_REQUESTED** [review]
Apart from a minor improvement, please update the `lib/sql` submodule before merging :)

The rest is looking good!

File: app/models/concerns/cartodb_central_synchronizable.rb (hunk 1 above)
Comment (juanrmn) [reply app/models/concerns/cartodb_central_synchronizable.rb:83]: You're totally right, I had this one as Draft: :facepalm: : https://github.com/CartoDB/cartodb-central/pull/2992

File: lib/carto/subscribers/map_views_update.rb (hunk 2 above)
Comment (juanrmn) [reply lib/carto/subscribers/map_views_update.rb]: Done.

**Review by thedae — APPROVED** [review]
:+1:

**Pull request merged**